### PR TITLE
WIP: Updated code

### PR DESCRIPTION
**Pull request by DanielTurner**
Since we are working agile the code is always in a WIP state until the project is deemed done.

* Updated all the javascript code
* all json/ files for form for backup (Added)
* package.json (Updated)
* Added postinstall to keep the parceljs working (Added)
* button-group.js manages the navigation (Added)
* formio-wrapper.js - This will be moved to NPM once we are happy with the included functionality this de-couples this from formio. (Added)
* topics-list.js - Controls the topics. (Added)
* index.html & index.js - is the app shell (Updated)
* styles - (Updated)
* resources/ - embedded html inside of formio (Updated & Added)
* test/ - Unit tests covering javascript functionality.

### CODE REVIEW

**Comment by DanielTurner** [comment]
This doesn't relate to any one ticket as the tickets assume full functionality.

File: src/components/topics-list.js [hunk 1]
```diff
@@ -0,0 +1,274 @@
...
+              'https://api.forms.platforms.qld.gov.au/fesrqwsyzlbtegd/healthandhygiene',
+            image: 'https://www.qld.gov.au/?a=140674',
+          },
+        ],
+      },
+      {
+        name: 'Process',
+        topics: [
+          {
+            title: 'Recieve food',
```
Comment (ThrawnCA) [inline src/components/topics-list.js:32]: Misspelled "Receive"

File: src/components/topics-list.js [hunk 2]
```diff
@@ -0,0 +1,274 @@
...
+            form:
+              'https://api.forms.platforms.qld.gov.au/fesrqwsyzlbtegd/recievefood',
```
Comment (ThrawnCA) [inline src/components/topics-list.js:34]: Misspelled "receive"

File: src/components/topics-list.js [hunk 3]
```diff
@@ -0,0 +1,274 @@
...
+        name: 'Premises and equipment',
+        topics: [
+          {
+            title: 'Animals and pests',
+            form:
+              'https://api.forms.platforms.qld.gov.au/fesrqwsyzlbtegd/animalsandpests',
+            image: 'https://www.qld.gov.au/?a=140677',
+          },
+          {
+            title: 'Clean and santise',
```
Comment (ThrawnCA) [inline src/components/topics-list.js:79]: Misspelled "sanitise"

File: src/resources/overview/view.html [hunk 4]
```diff
@@ -15,7 +15,13 @@ <h1>Animals and pests</h1>
 </p>
 
 <p>
-    Only dogs are allowed in outdoor dining areas if you as the business choose to allow them. Check with your <a href="https://www.qld.gov.au/about/how-government-works/local-government-directory">local council</a> for the rules that apply in your area.
+    Only dogs are allowed in outdoor dining areas if you as the business choose to allow them. Check with your
```
Comment (ThrawnCA) [inline src/resources/overview/view.html:18]: "Only dogs are allowed" should be "Dogs are only allowed"

File: src/resources/questions-health-hygiene/question-8/no.html [hunk 5]
```diff
@@ -0,0 +1,35 @@
+<div class="alert alert-info" role="alert">
+  <h2><i class="fa fa-exclamation-triangle"></i>Warning!</h2>
+  <p>
+   A staff illness policy or procedure helps you to:
```
Comment (ThrawnCA) [inline src/resources/questions-health-hygiene/question-8/no.html:4]: This sentence is incomplete.

File: src/resources/summary/animal-pests-section1/view.html [hunk 6]
```diff
@@ -0,0 +1,117 @@
+<link rel="stylesheet" href="./styles.scss">
+
+<h1>Summary</h1>
+
+<p>
+    Thank you for taking the time to complete the Animals and pests
```
Comment (ThrawnCA) [inline src/resources/summary/animal-pests-section1/view.html:6]: Inconsistent capitalisation.

**Review by ThrawnCA — CHANGES_REQUESTED** [review]

File: src/resources/home/view.html [hunk 7]
```diff
@@ -28,15 +28,15 @@ <h3>People</h3>
     <section class="row qg-cards">
         <article class="qg-card col-12 col-sm-6 col-lg-4">
             <div class="content">
-                <img src="https://via.placeholder.com/190x186" alt="">
+                <img src="https://www.qld.gov.au/dev/food-pantry/dev/kyfb-home-images/Skills-and-knowledge-176606698.jpg" alt="">
```
Comment (duttonw) [inline src/resources/home/view.html:31]: unsure, may want to be relative or prd version of images

**Review by duttonw — APPROVED** [review]
Will merge, please correct what has been raised

**Pull request merged**